### PR TITLE
docs(refactor): tutorials MTA-1809

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

File: tutorials/encrypt-s3-data-rclone/index.mdx [hunk 1]
```diff
@@ -20,7 +20,7 @@ dates:
 
 Rclone is available for Windows, macOS X and various Linux distributions.
 
-In this tutorial we will have a look on the [rlcone crypt](https://rclone.org/crypt/) module to encrypt your data before sending it to [Scaleway Object Storage](https://www.scaleway.com/en/object-storage/) via the S3 protocol.
+In this tutorial we have a look on the [rlcone crypt](https://rclone.org/crypt/) module to encrypt your data before sending it to [Scaleway Object Storage](https://www.scaleway.com/en/object-storage/) via the S3 protocol.
```
Comment (RoRoJ) [inline tutorials/encrypt-s3-data-rclone/index.mdx:23]:
```suggestion
In this tutorial we have a look at the [rlcone crypt](https://rclone.org/crypt/) module to encrypt your data before sending it to [Scaleway Object Storage](https://www.scaleway.com/en/object-storage/) via the S3 protocol.
```

File: tutorials/foreman-puppet/index.mdx [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
 
 Foreman helps to automatize the OS installation. After that – through its very good integration with puppet – the new system will be configured towards the desired state. Finally Puppet will send facts about the system to Foreman which helps to monitor the whole system over its complete lifecycle. With a discovery plugin Foreman can also discover new machines in the network based on their mac address.
 
-This tutorial assumes that Foreman is being installed on a fresh instance, which will also act as the Puppet master server. Foreman requires a working Puppet installation to perform the configuration management of hosts.
+This tutorial assumes that Foreman is being installed on a fresh instance, which will also act as the Puppet primary server. Foreman requires a working Puppet installation to perform the configuration management of hosts.
```
Comment (RoRoJ) [inline tutorials/foreman-puppet/index.mdx:23]:
```suggestion
This tutorial assumes that Foreman is being installed on a fresh Instance, which will also act as the Puppet primary server. Foreman requires a working Puppet installation to perform the configuration management of hosts.
```

File: tutorials/foreman-puppet/index.mdx [hunk 3]
```diff
@@ -45,11 +45,11 @@ The installation tool is a collection of Puppet modules that installs everything
   ssh root@SERVER_IP
   ```
 
-  To retrieve the Instances IP, list your exiting servers using `scw ps` (Scaleway CLI). For more information on the Scaleway CLI, refer to the tutorial on the [Scaleway Command Line Interface](https://www.scaleway.com/docs/manage-cloud-servers-with-scaleway-cli/).
+  To retrieve the Instances IP, list your exiting servers using `scw ps` (Scaleway CLI). For more information on the Scaleway CLI, refer to the tutorial on the [Scaleway Command Line Interface](https://github.com/scaleway/scaleway-cli/).
 
   The Instances IP can also be retrieved from the Scaleway Console. Once logged in, check the **IP Adresses** in the **Instances** tab of the left menu.
 
-2. Check your Instances full hostname to ensure the hostname is set to [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name)
+2. Check your Instances full hostname to ensure the hostname is set to a valid [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name).
```
Comment (RoRoJ) [inline tutorials/foreman-puppet/index.mdx:52]:
```suggestion
2. Check your Instance's full hostname to ensure the hostname is set to a valid [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name).
```

File: tutorials/foreman-puppet/index.mdx [hunk 4]
```diff
@@ -130,13 +130,13 @@ The fastest way to install Foreman is by using the installer, which installs and
...
     * Foreman Proxy is running at https://foreman.example.com:8443
-    * Puppetmaster is running at port 8140
+    * Puppetprimary is running at port 8140
     The full log is at /var/log/foreman-installer/foreman.log
   ```
 
 ## Managing Puppet
 
-During installation, the installation tool will sets up a puppet master on the host, which is fully integrated with Foreman. Run the Puppet agent on the instance to send the first Puppet report to Foreman, This creates the host automatically in Foremans database.
+During installation, the installation tool will sets up a puppet primary on the host, which is fully integrated with Foreman. Run the Puppet agent on the instance to send the first Puppet report to Foreman, This creates the host automatically in Foremans database.
```
Comment (RoRoJ) [inline tutorials/foreman-puppet/index.mdx:139]:
```suggestion
During installation, the installation tool will sets up a puppet primary on the host, which is fully integrated with Foreman. Run the Puppet agent on the Instance to send the first Puppet report to Foreman, This creates the host automatically in Foremans database.
```

File: tutorials/host-gitlab-runner/index.mdx [hunk 5]
```diff
@@ -6,34 +6,23 @@ content:
...
 
-GitHub provides a limited range of free resources to build your applications through GitHub Actions and professional developers can reach their limits quickly. Besides, teams working on professional projects tend to prefer having full control over their build environment. GitHub provides the possibility to execute runners on self-managed Instances. A runner is an application that runs a job from a GitHub Actions workflow.
-
-In this tutorial, you will learn how to set up a GitHub Actions runner on a Scaleway Elements Instance to automatize the workflow of your projects. For demanding workloads and best performances, the use of [General Purpose](https://www.scaleway.com/en/virtual-instances/general-purpose/) Instances is recommended.
+GitHub provides a limited range of free resources to build your applications through GitHub Actions, and professional developers can reach their limits quickly. Besides, teams working on professional projects prefer having complete control over their build environment. GitHub provides the possibility to execute runners on self-managed Instances. A runner is an application that runs a job from a GitHub Actions workflow.
```
Comment (RoRoJ) [inline tutorials/host-gitlab-runner/index.mdx:18]:
```suggestion
GitHub provides a limited range of free resources to build your applications through GitHub Actions, and professional developers can reach these limits quickly. Teams working on professional projects may also prefer to have complete control over their build environment. GitHub provides the possibility to execute runners on self-managed Instances. A runner is an application that runs a job from a GitHub Actions workflow.
```

File: tutorials/deploy-saas-application/index.mdx [hunk 6]
```diff
@@ -6,48 +6,29 @@ content:
...
+In this tutorial you will learn how to deploy a Software as-a-Service (SaaS) application, based on a SaaS application developed using Django Template. You use the following Scaleway products:
 
 * [**Object Storage**](https://www.scaleway.com/en/object-storage/), to store your static files
 * [**Database as-a-Service**](https://www.scaleway.com/en/database/), for your data storage
 * [**Container Registry**](https://www.scaleway.com/en/container-registry/), to store your Docker images
 * [**Kubernetes Kapsule**](https://www.scaleway.com/en/kubernetes-kapsule/), for your production environment
 
-Along the way, you'll learn how to store environment variables with Kubernetes secrets and use ImagePullSecrets to pull your private images. We also give you some tips for going further with your deployed SaaS application, including how to expose it with a Scaleway Load Balancer.
+Along the way, you learn how to store environment variables with Kubernetes secrets and use ImagePullSecrets to pull your private images. We also give you some tips for going further with your deployed SaaS application, including how to expose it with a Scaleway Load Balancer.
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-saas-application/index.mdx:25]:
```suggestion
You will learn how to store environment variables with Kubernetes secrets and use ImagePullSecrets to pull your private images. We also give you some tips for going further with your deployed SaaS application, including how to expose it with a Scaleway Load Balancer.
```

**Review by RoRoJ — APPROVED** [review]

File: tutorials/encrypt-s3-data-rclone/index.mdx [hunk 7]
```diff
@@ -20,7 +20,7 @@ dates:
 
 Rclone is available for Windows, macOS X and various Linux distributions.
 
-In this tutorial we will have a look on the [rlcone crypt](https://rclone.org/crypt/) module to encrypt your data before sending it to [Scaleway Object Storage](https://www.scaleway.com/en/object-storage/) via the S3 protocol.
+In this tutorial we have a look at the [rlcone crypt](https://rclone.org/crypt/) module to encrypt your data before sending it to [Scaleway Object Storage](https://www.scaleway.com/en/object-storage/) via the S3 protocol.
```
Comment (ldecarvalho-doc) [inline tutorials/encrypt-s3-data-rclone/index.mdx:23]:
```suggestion
In this tutorial we use the [rlcone crypt](https://rclone.org/crypt/) module to encrypt your data before sending it to [Scaleway Object Storage](https://www.scaleway.com/en/object-storage/) via the S3 protocol.
```

File: tutorials/foreman-puppet/index.mdx [hunk 8]
```diff
@@ -1,39 +1,39 @@
...
 
 Foreman helps to automatize the OS installation. After that – through its very good integration with puppet – the new system will be configured towards the desired state. Finally Puppet will send facts about the system to Foreman which helps to monitor the whole system over its complete lifecycle. With a discovery plugin Foreman can also discover new machines in the network based on their mac address.
 
-This tutorial assumes that Foreman is being installed on a fresh instance, which will also act as the Puppet master server. Foreman requires a working Puppet installation to perform the configuration management of hosts.
+This tutorial assumes that Foreman is being installed on a fresh Instance, which will also act as the Puppet primary server. Foreman requires a working Puppet installation to perform the configuration management of hosts.
```
Comment (ldecarvalho-doc) [inline tutorials/foreman-puppet/index.mdx:23]: To me this should be a requirement

File: tutorials/foreman-puppet/index.mdx [hunk 9]
```diff
@@ -130,13 +130,13 @@ The fastest way to install Foreman is by using the installer, which installs and
...
     * Foreman Proxy is running at https://foreman.example.com:8443
-    * Puppetmaster is running at port 8140
+    * Puppetprimary is running at port 8140
     The full log is at /var/log/foreman-installer/foreman.log
   ```
 
 ## Managing Puppet
 
-During installation, the installation tool will sets up a puppet master on the host, which is fully integrated with Foreman. Run the Puppet agent on the instance to send the first Puppet report to Foreman, This creates the host automatically in Foremans database.
+During installation, the installation tool will sets up a puppet primary on the host, which is fully integrated with Foreman. Run the Puppet agent on the Instance to send the first Puppet report to Foreman, This creates the host automatically in Foremans database.
```
Comment (ldecarvalho-doc) [inline tutorials/foreman-puppet/index.mdx:139]:
```suggestion
During installation, the installation wizard sets up a puppet primary on the host, which is fully integrated with Foreman. Run the Puppet agent on the Instance to send the first Puppet report to Foreman, This creates the host automatically in Foremans database.
```

File: tutorials/foreman-puppet/index.mdx [hunk 10]
```diff
@@ -1,39 +1,44 @@
...
 
 Foreman helps to automatize the OS installation. After that – through its very good integration with puppet – the new system will be configured towards the desired state. Finally Puppet will send facts about the system to Foreman which helps to monitor the whole system over its complete lifecycle. With a discovery plugin Foreman can also discover new machines in the network based on their mac address.
 
-This tutorial assumes that Foreman is being installed on a fresh instance, which will also act as the Puppet master server. Foreman requires a working Puppet installation to perform the configuration management of hosts.
+<<<<<<< HEAD
```
Comment (bene2k1) [inline tutorials/foreman-puppet/index.mdx:23]:
```suggestion
```

File: tutorials/foreman-puppet/index.mdx [hunk 11]
```diff
@@ -1,39 +1,44 @@
...
+This tutorial assumes that Foreman is being installed on a fresh instance, which will also act as the Puppet primary server. 
+=======
```
Comment (bene2k1) [inline tutorials/foreman-puppet/index.mdx:25]:
```suggestion
```

File: tutorials/foreman-puppet/index.mdx [hunk 12]
```diff
@@ -1,39 +1,44 @@
...
+This tutorial assumes that Foreman is being installed on a fresh Instance, which will also act as the Puppet primary server. Foreman requires a working Puppet installation to perform the configuration management of hosts.
```
Comment (bene2k1) [inline tutorials/foreman-puppet/index.mdx:26]:
```suggestion
```

**Pull request merged**